### PR TITLE
Swagger fixes issue #194

### PR DESCRIPTION
**Pull request by vanderaj**
This pull request fixes all of the noted issues in #194 , which is important for those generating type safe EliteBGS clients

* Fixes the influence to be a double float instead of an integer, which is what it is internally
* Fixes the x, y, z to be double float instead of integers, which is what it is internally 
* Fixes the population response types to be a 64-bit integer, which is what it is internally

Additionally, this PR:

* Removes all vestiges of the deprecated/removed V1 ... V4 API
* Fixes the Swagger schemes, so it now passes validation in several tools
* Formally moves tickv4 to tickv5 externally (although it still maintains the v4 collection on MongoDB)

Additional improvements could be to migrate the tickv4 MongoDB collection to tickv5

### CODE REVIEW

**Comment by vanderaj** [comment]
One last comment is that this PR also contains a .prettierrc file to allow me to work on Windows. GitHub automatically translates files into CRLF on this platform and vice versa, but Prettier doesn't accept this. So my change to that one file is to permit CRLF or standard Unix automatically.

**Comment by vanderaj** [comment]
Example of the v5 API now passing validation:

```
PS C:\Users\Andrew van der Stock\Documents\GitHub\go-swagger> swagger validate .\elitebgs-v5.json
2021/05/06 14:16:29 
The swagger spec at ".\\elitebgs-v5.json" is valid against swagger specification 2.0
```

and one of the generated outputs:

```
	// population
	Population int64 `json:"population,omitempty"`
...
	// x
	X float64 `json:"x,omitempty"`

	// y
	Y float64 `json:"y,omitempty"`

	// z
	Z float64 `json:"z,omitempty"`
}
```

**Comment by vanderaj** [comment]
Found issues with eddn_listener and older versions. I need to make sure we're only updating one of the models.

**Comment by vanderaj** [comment]
Okay, I've fixed and tested the whole lot. I'm taking it out of draft. It's ready for review. Current suggestions:

Remove most of journal.js in eddn_listener - it's dead code, but I'm happy to wait until you've reviewed this PR
Do a migration of the collection ticktimesv4 to ticktimesv5 and associated changes in backend, eddn_listener, and tick_listener.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (SayakMukhopadhyay) [inline .prettierrc:1]: Is a top level prettierrc necessary?

File: backend/.prettierrc [hunk 2]
```diff
@@ -3,5 +3,8 @@
   "singleQuote": true,
   "endOfLine": "lf",
   "trailingComma": "none",
-  "printWidth": 120
+  "printWidth": 120,
+  ["error", {
```
Comment (SayakMukhopadhyay) [inline backend/.prettierrc:7]: This is showing me invalid json

**Review by SayakMukhopadhyay — CHANGES_REQUESTED** [review]
The changes to the EDDN listener journal file is kind of not needed from your end. I was planning to delete it anyway. So, give me a moment while I delete that code and then you can rebase this PR onto the master.

**Comment by SayakMukhopadhyay** [comment]
> 
> 
> One last comment is that this PR also contains a .prettierrc file to allow me to work on Windows. GitHub automatically translates files into CRLF on this platform and vice versa, but Prettier doesn't accept this. So my change to that one file is to permit CRLF or standard Unix automatically.

I am using Windows myself and I too pretty much always have to run `npm run lint:fix` after a checkout but I got this pragmatic approach after I faced an issue where a deployed application wouldn't run because some obscure file's line ending was not set to unix. Still, the `.prettierrc` file doesn;t seem to work for me so can't test if it suits my workflow.

**Comment by SayakMukhopadhyay** [comment]
This PR is dependant on #231 being closed which I will do ASAP post which this PR might need some fixing

**Comment by SayakMukhopadhyay** [comment]
#231 is closed but you will need to rebase this PR on master and there will be conflicts. Please accept the changes from my side while resolving the conflict.

File: backend/.prettierrc (hunk 2 above)
Comment (vanderaj) [reply backend/.prettierrc:7]: Removed

> 
> 
> The changes to the EDDN listener journal file is kind of not needed from your end. I was planning to delete it anyway. So, give me a moment while I delete that code and then you can rebase this PR onto the master.

I've deleted this file and checked with a no-commit that it should merge without issues. Ready for re-review

**Review by SayakMukhopadhyay — APPROVED** [review]
There are a few things here and there but this is ok to merge for now.

**Pull request merged**